### PR TITLE
Add option to keep keyboard letters uppercase

### PR DESCRIPTION
**Pull request by TheInterestingSoul**
Screenshot_20220914_090532_com coolapk market

### CODE REVIEW

**Comment by TheInterestingSoul** [comment]
```
 > Connect to repo.maven.apache.org:443 [repo.maven.apache.org/146.75.28.215] failed: Connect timed out
```

action因为自身网络问题停了，能不能重启一下，谢谢，我这边是能编译过的

File: app/src/main/java/org/fcitx/fcitx5/android/input/keyboard/TextKeyboard.kt [hunk 1]
```diff
@@ -179,9 +180,15 @@ class TextKeyboard(
         }
     }
 
+    private var keyboardLettersRemainUppercase by AppPrefs.getInstance().keyboard.keyboardLettersRemainUppercase
```
Comment (berberman) [inline app/src/main/java/org/fcitx/fcitx5/android/input/keyboard/TextKeyboard.kt:183]: Just a nitpick: use `val` if the value doesn't need to be changed

File: app/src/main/res/values-zh-rCN/strings.xml [hunk 2]
```diff
@@ -145,4 +145,5 @@
     <string name="skip">跳过</string>
     <string name="horizontal_candidate_growth">拉伸横向候选词以填充宽度</string>
     <string name="system_touch_sounds">系统按键音</string>
+    <string name="keyboard_letters_remain_uppercase">键面字母保持大写</string>
```
Comment (berberman) [inline app/src/main/res/values-zh-rCN/strings.xml:148]: Please don't change this file, as it was generated by transifex

**Review by berberman — CHANGES_REQUESTED** [review]

**Review by berberman — APPROVED** [review]
Thank you!

**Pull request merged**